src/social-network/worker.ts

```diff
@@ -22,36 +25,45 @@ export interface SocialNetworkWorker extends SocialNetworkWorkerAndUI {
      *
      * If this function is not provided, autoVerifyBio in Welcome will be unavailable
      */
-    autoVerifyBio?(user: PersonIdentifier, provePost: string): void
+    autoVerifyBio?: ((user: PersonIdentifier, provePost: string) => void) | null
```

Comment:
With `?` we don't really have to do this; `?` implies `| undefined`, which is good enough for specifying a "not available" value IMO.

Reply:
The thing is, ```undefined``` means undefined (at definition time), everything must be defined at runtime (for the convenient of managing config items), so the ```null``` explicit defines "not available".

Reply:
This will be written to comment at feat/twitter3